packages/core/src/Provider/Provider.js

```diff
@@ -182,32 +183,18 @@ HvProvider.propTypes = {
      */
     seed: PropTypes.string,
   }),
-  /**
-   * Injects the generated stylesheets at the top of the `<head>` element of the page.
-   * This can ease the override of UI Kit components styles.
-   *
-   * By default, the styles are injected last in the `<head>` element of the page.
-   */
-  injectStylesFirst: PropTypes.bool,
   /**
    * Disables the generation of the styles.
    */
   disableStylesGeneration: PropTypes.bool,
-
   /**
-   * Disables the generation of the baseline css styles.
-   *
-   * This will be the default behavior in the future.
-   *
-   * The application using UI Kit should be responsible for adding the baseline css styles, by
-   * either using the `<HvCssBaseline />` component, using the `<HvScopedCssBaseline />` component,
-   * or ensuring that the necessary base styles are applied.
-   *
-   * Defaults to `false`. Will be removed in the next major release.
+   * By default the baseline styles are applied globally to the application.
+   * If you need to scope the CSS to avoid styling conflicts, you can set this prop to `"scoped"`.
+   * If you are providing the baseline styles, you can set this prop to false.
    *
-   * @see https://lumada-design.github.io/uikit/master/?path=/docs/foundation-css-baseline--main
+   * @see https://lumada-design.github.io/uikit/master/?path=/docs/theme-css-baseline--page
    */
-  disableCssBaseline: PropTypes.bool,
+  cssBaseline: PropTypes.oneOf(["global", "scoped", false]),
```

Comment:
Maybe instead of false we could use "none" so it is more inline with the enum typing instead of enum + bool